example/py-shiny/example_pyshiny_scatter.py

```diff
@@ -0,0 +1,38 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+from maidr.custom_renderers.pyshiny_renderer import render_maidr
+from shiny import App, render, ui
+
+# Load the datasets
+penguins = sns.load_dataset("penguins")
+iris = sns.load_dataset("iris")
+
+# Define the UI for the application
+app_ui = ui.page_fluid(ui.row(ui.column(12, ui.output_ui("scatterplot"))))
+
+
+# Define the server logic
+def server(input, output, session):
+    def create_scatterplot():
+        fig, ax = plt.subplots(figsize=(10, 6))
+        s_plot = sns.scatterplot(
+            data=iris, x="sepal_length", y="sepal_width", hue="species", ax=ax
+        )
+        ax.set_title("Iris Sepal Length vs Sepal Width")
+        ax.set_xlabel("Sepal Length")
+        ax.set_ylabel("Sepal Width")
+        return s_plot
+
+    @output
+    @render.ui
+    def scatterplot():
+        return render_maidr(create_scatterplot)()
+
+
```

Comment:
This is too simple. Demonstrate the example Shiny app where users can choose two different numeric variables (one for x; the other for y) from the iris dataset and update the plot reactively against the variable change.

Reply:
Sure, Professor. I will create a more complex example to demonstrate the feature more intuitively.